src/core/lib/iomgr/tcp_server_windows.cc

```diff
@@ -255,7 +255,7 @@ static grpc_error* start_accept_locked(grpc_tcp_listener* port) {
   }
 
   sock = WSASocket(AF_INET6, SOCK_STREAM, IPPROTO_TCP, NULL, 0,
-                   WSA_FLAG_OVERLAPPED);
+                   WSA_FLAG_OVERLAPPED | WSA_FLAG_NO_HANDLE_INHERIT);
```

Comment:
It looks like there are more places where WSASocket is created. Any reason why you're only updating occurrences in this file?

https://github.com/grpc/grpc/search?q=WSASocket&unscoped_q=WSASocket

Reply:
I'm not sure of the semantics and mechanics used elsewhere and have only tested this server-side diff with some long-running stress tests. I think it's a good idea to set this flag in src/core/lib/iomgr/tcp_client_windows.cc and other places unless they imply using some kind of magic to pass a socket to child processes for some reason. But I'm afraid I won't be able to verify them properly. 
I can update the review placing WSA_FLAG_NO_HANDLE_INHERIT everywhere, if you rely on automated tests.

Reply:
> @frazenshtein is this PR an attempt to resolve #17704?
> Have you been able to confirm that it fixes the problem?

I believe this PR will fix this issue. In my case, the problem looked like an error on the client side: `grpc on_accept error: the I/o operation was aborted due to the completion of the command flow or the request of the application.`

Reply:
Btw this flag is used by default in the go: https://golang.org/src/net/sock_windows.go line 21